dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/ResponseData.java

```diff
@@ -32,7 +32,8 @@ private void setIdAttribute(Object obj) {
       Field field = FieldUtil.findDeclaredField(obj, "id");
       field.setAccessible(true);
       field.set(obj, null);
-    } catch (NoSuchFieldException | SecurityException | IllegalArgumentException | IllegalAccessException e) {
+    } catch (NoSuchFieldException | SecurityException | IllegalArgumentException | IllegalAccessException | NullPointerException e) {
+      //catch a NPE here if we have a null attributes section on purpose due to no values in the attributes section
       e.printStackTrace();
```

Comment:
Is this a situation that we will have to address manually? If so, we should log it as ERROR so that we get an email. Log.error (if passed with the exception as an argument) will automatically print the stack trace.